e2e/full_scan_test.go

```diff
@@ -183,3 +196,91 @@ var _ = ginkgo.Describe("Running a full scan (exploits, info finder, malware, mi
 		}
 	})
 })
+
+// generateMarkdownTable generates a markdown table using the provided scan stats
+// and writes it to a file, that will be displayed in github summary during CI run.
+func generateMarkdownTable(scanStats *apitypes.AssetScanStats) string {
```

Comment:
suggestion: lets move all benchmarking code to 'e2e/scan_stats.go'. the stats logic can be a struct with private methods and '.GenerateStats' method that glues everything together and returns a string for the generated table